src/Features/VisualBasic/Portable/ExtractMethod/VisualBasicSelectionValidator.vb

```diff
@@ -494,6 +494,18 @@ result.ReadOutside().Any(Function(s) Equals(s, local)) Then
                        }
             End If
 
+            Dim commonRootContainer = commonRoot.FirstAncestorOrSelf(Of SyntaxNode)(Function(node) node.Kind = SyntaxKind.ClassBlock Or node.Kind = SyntaxKind.StructureBlock Or node.Kind = SyntaxKind.ModuleBlock)
```

Comment:
Isn't `Kind` here a method? If so, I think better to change it to `Kind()` or use `IsKind()`.

Reply:
forgot about isKind, thanks!